### PR TITLE
Add React-PDF

### LINKED ISSUE #10471: Please add an icon for React-PDF.
### Brand Name

React-PDF

### Website

https://react-pdf.org/

### Popularity Metric

The `React-PDF` popularity metric is **543,979**

See [https://www.similarweb.com/website/react-pdf.org/](https://www.similarweb.com/website/react-pdf.org/)

### Official Resources for Icon and Color

png-file resource: [https://react-pdf.org/images/logo.png](https://react-pdf.org/images/logo.png)

colors:
- #ed0000
- #e90000
- #bc0200
- #820100

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by nirvikpurkait**
reactpdf

**Issue:** closes #10471

**Similarweb rank:**
543,979

**GitHub star:**
13.9k

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

- png converted to svg with inkscape
- had multiple colors, all are red with slightly variation, so mixed all red colors.

### CODE REVIEW

**Review by PeterShaggyNoble — COMMENTED** [review]
Thanks for the contribution, @nirvikpurkait 👍🏻 However, I think we'll need some outlines around the top layer of shapes in order to maintain the integrity of and illusion of depth from the original. Would you be able to give that a try, please? Unfortunately, I can't find a vector version of the original, either.

**Comment by nirvikpurkait** [comment]
Hey @PeterShaggyNoble , thanks for response. 

Ummm, I am not getting what are you asking for. Are you asking for something like this?

Frame 3

**Comment by PeterShaggyNoble** [comment]
If you take that white outline you have around the 3 shapes at the front, increase its width to 0.5dp and then cut it out of the main shape behind them, that should do the trick.

**Comment by nirvikpurkait** [comment]
Hello @PeterShaggyNoble, I am facing an error. How can I get rid of it?

```shell
  x icon-size Size of <path> must be exactly 24 in one dimensi    
              on; the size is currently 23.999 x 21.571
  x icon-centered <path> must be centered at (12, 12); the cen    
                  ter is currently (12, 11.996)
```

**Comment by adamrusted** [comment]
It looks like it's having an issue optimising the new path to 3 decimals. How're you optimising it? If you're using `npx svgo` you can add the flag `-p 4` to up the precision to 4 decimals. That might resolve your issue!

**Comment by nirvikpurkait** [comment]
> It looks like it's having an issue optimising the new path to 3 decimals. How're you optimising it? If you're using `npx svgo` you can add the flag `-p 4` to up the precision to 4 decimals. That might resolve your issue!

@adamrusted thank you for the advise, I was using npx svgo to optimize the svg, but didn't know about the flag, so when ever I optimize the svg it gives error. 

How ever I successfully optimized the svg, and the outcome looks like this (with color for visualization) ,
Frame

Is this the result you wanted @PeterShaggyNoble ? Or is there anything else we can improve?

**Comment by nirvikpurkait** [comment]
image

**Comment by adamrusted** [comment]
@nirvikpurkait - please don't feel the need to merge develop into this branch, it notifies everyone that has commented every time you do it. We'll merge develop into this when we come to review it.

**Comment by nirvikpurkait** [comment]
> @nirvikpurkait - please don't feel the need to merge develop into this branch, it notifies everyone that has commented every time you do it. We'll merge develop into this when we come to review it.

Ohh, sorry, my bad. I thought that I always need to keep my branch updated. I won't do it next time.

**Comment by adamrusted** [comment]
> Ohh, sorry, my bad. I thought that I always need to keep my branch updated.

No worries! It's good practice to keep your 'develop' branch up to date with ours - just so any new PRs you submit don't have any merge issues, but once you've submitted a PR you're fine to leave it unless you're changing somethin 😄

**Comment by nirvikpurkait** [comment]
Sorry, I accidentally update this branch instead of another branch

**Comment by nirvikpurkait** [comment]
Hello, @PeterShaggyNoble , @adamrusted if everything is fine, can you add this to the develop branch as this would motivate me to take on other issues too. 

Also I need this logo too, to add somewhere else.

Thank you.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -12401,6 +12401,11 @@
             "hex": "FF4154",
             "source": "https://github.com/tannerlinsley/react-table/blob/8c77b4ad97353a0b1f0746be5b919868862a9dcc/docs/src/images/emblem-light.svg"
         },
+        {
+            "title": "React-PDF",
+            "hex": "B4080A",
+            "source": "https://react-pdf.org/images/logo.png"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:12407]: Also, just the website itself will do for the `source` URL.
```suggestion
            "source": "https://react-pdf.org"
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
It definitely works much better with the cut-outs, @nirvikpurkait, nice work. There are, however, a few issues which will need to be corrected before we can merge this.

1. Your version doesn't quite line up with the source (see comparison below, with yours in green),
2. The cut-outs should knocked out of the bottom layered shape (i.e., overlaying the shadows of the upper layers) and should be 0.5dp, and,
3. There are quite a few excess points which could be cleaned up.

**Comment by nirvikpurkait** [comment]
image

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
That lines up nicely now, @nirvikpurkait 👍🏻

However, there are still a number of excess points that could be removed. Would you be able to give that a try, please?

**Pull request closed without merging**